### PR TITLE
Add 1.0 release info to timeline

### PR DESCRIPTION
**Pull request by rohitsardesai83**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Adds 1.0 release to timeline.

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: content/en/home/timeline.md [hunk 1]
```diff
@@ -18,9 +18,27 @@ date_format = "January 2006"
...
+- CRI-compliant runtimes like containerd are supported at the edge. This enables edged to communicate with the runtime to manage containers running on resource constrained edge nodes.
+- Enhanced cloud and edge communication efficiency via QUIC, a UDP-based protocol. CloudHub supports both Websocket and QUIC protocol access at the same time. The edgehub can choose one of the protocols to access the cloudhub. <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md' target="_blank">&hellip; Read more</a>
+- Modbus Mapper to connect and control devices that use Modbus(RTU/TCP) as a communication protocol.<a href='https://github.com/kubeedge/kubeedge/blob/master/docs/mappers/modbus_mapper.md'target="_blank">&hellip; Read more</a>
+- Edge Site enables to run a standalone Kubernetes cluster at the edge along with KubeEdge to get full control and improve the offline scheduling capability. <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/EdgeSite.md' target="_blank">&hellip; Read more</a>
+
+  """
```
Comment (anvithks) [inline content/en/home/timeline.md]: Remove extra new lines and double quotes.

File: content/en/home/timeline.md [hunk 2]
```diff
@@ -18,9 +18,27 @@ date_format = "January 2006"
...
+  title = "v1.0.0"
+  company = "KubeEdge v1.0.0 release"
+  company_url = "https://github.com/kubeedge/kubeedge/releases/tag/v1.0.0"
+  location = "June, 2019"
+  date_start = ""
+  date_end = ""
+  description = """
+**Features added**
+
+- Edge mesh provides service mesh capabilities on the edge. Pod-to-pod communication on the same edge node or across edge nodes in the same subnet is supported. <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/edgemesh-design.md' target="_blank">&hellip; Read more</a>
```
Comment (anvithks) [inline content/en/home/timeline.md]: Point the links to the appropriate urls from the docs.kubeedge.io website.

File: content/en/home/timeline.md [hunk 3]
```diff
@@ -18,9 +18,27 @@ date_format = "January 2006"
...
+- CRI-compliant runtimes like containerd are supported at the edge. This enables edged to communicate with the runtime to manage containers running on resource constrained edge nodes.
+- Enhanced cloud and edge communication efficiency via QUIC, a UDP-based protocol. CloudHub supports both Websocket and QUIC protocol access at the same time. The edgehub can choose one of the protocols to access the cloudhub. <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md' target="_blank">&hellip; Read more</a>
```
Comment (anvithks) [inline content/en/home/timeline.md]: Same as above.

File: content/en/home/timeline.md [hunk 4]
```diff
@@ -18,9 +18,27 @@ date_format = "January 2006"
...
+- Modbus Mapper to connect and control devices that use Modbus(RTU/TCP) as a communication protocol.<a href='https://github.com/kubeedge/kubeedge/blob/master/docs/mappers/modbus_mapper.md'target="_blank">&hellip; Read more</a>
+- Edge Site enables to run a standalone Kubernetes cluster at the edge along with KubeEdge to get full control and improve the offline scheduling capability. <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/EdgeSite.md' target="_blank">&hellip; Read more</a>
```
Comment (anvithks) [inline content/en/home/timeline.md]: Same as above. Please update the links to the appropriate urls from the docs website.
Proposals is internal and not available in a readable format. If this needs to be shown to the users then move the proposals file to the docs folder.

File: content/en/home/timeline.md (hunk 1 above)
Comment (rohitsardesai83) [reply content/en/home/timeline.md]: I guess these are required for multi-line descriptions. Removed the new line.

File: content/en/home/timeline.md (hunk 2 above)
Comment (rohitsardesai83) [reply content/en/home/timeline.md]: updated with the website links for available documentation. removed md links.

File: content/en/home/timeline.md (hunk 3 above)
Comment (rohitsardesai83) [reply content/en/home/timeline.md]: done.

File: content/en/home/timeline.md (hunk 4 above)
Comment (rohitsardesai83) [reply content/en/home/timeline.md]: done.

File: content/en/home/timeline.md (hunk 1 above)
Comment (rohitsardesai83) [reply content/en/home/timeline.md]: I guess these are required for multi-line descriptions. Removed the new line.

Comment (anvithks) [reply content/en/home/timeline.md]: Oh right! The triple quotes are for the description as per the theme.

**Comment by anvithks** [comment]
@rohitsardesai83 Please avoid force pushing changes. The history is lost and there is no way to track why the changes were requested. 

/lgtm

**Pull request merged**